composer.json

```diff
@@ -21,40 +21,37 @@
     "extra": {
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ~8.0.0",
         "laminas/laminas-console": "^2.6",
```

Comment:
Drop 8.0 in this PR as well, since it is not installable with `laminas-console`

That won't matter for composer but It will make clear for users that they need to upgrade to 4.0 for php 8

Reply:
I think the same strategy than laminas-cache can be used in this case. example

in composer.json
````json
    "replace": {
        "llaminas/laminas-console": "*"
    }
````

in this way, 3.5.x can be use with php 8, dont you think it wort it ?

Reply:
I do not see a value in this. It is a dev dependency kind of package. Something else depending on it is quite unlikely.